libraries/AP_InertialSensor/AP_InertialSensor_Logging.cpp

```diff
@@ -193,10 +193,7 @@ void AP_InertialSensor::write_notch_log_messages() const
                 notches[0]);
         }
 
-        // ask the HarmonicNotchFilter object for primary gyro to
-        // log the actual notch centers
-        const uint8_t primary_gyro = AP::ahrs().get_primary_gyro_index();
-        notch.filter[primary_gyro].log_notch_centers(i, now_us);
+        notch.filter[AP::ahrs().get_primary_gyro_index()].log_notch_centers(i, now_us);
```

Comment:
This seems like the same code? I like the original format with the comment better.

Reply:
It will change again when the primary gyro change goes in, but happy to do that on the merge rather than here

Reply:
Fixed